.editorconfig

```diff
@@ -1,10 +1,14 @@
 root = true
 
 [*]
-indent_style = tab
-indent_size = 3
-tab_width = 3
-tab_size = 3
+indent_size = 2
+tab_width = 2
+tab_size = 2
+indent_style = space
```

Comment:
Ideally, we should follow some common conventions.
For example, https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md
Also, it would be nice to have automated code formatting,